docs/en/latest/plugins/consumer-restriction.md

```diff
@@ -33,14 +32,14 @@ The `consumer-restriction` Plugin allows users to set access restrictions based
 
 ## Attributes
 
-| Name               | Type          | Required | Default       | Valid values                                                                              | Description                                                                    |
-|--------------------|---------------|----------|---------------|-------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------|
-| type               | string        | False    | consumer_name | ["consumer_name", "service_id", "route_id"]                                               | Type of object to base the restriction on.                                     |
-| whitelist          | array[string] | True     |               |                                                                                           | List of objects to whitelist. Has a higher priority than `allowed_by_methods`. |
-| blacklist          | array[string] | True     |               |                                                                                           | List of objects to blacklist. Has a higher priority than `whitelist`.          |
-| rejected_code      | integer       | False    | 403           | [200,...]                                                                                 | HTTP status code returned when the request is rejected.                        |
-| rejected_msg       | string        | False    |               |                                                                                           | Message returned when the request is rejected.                                 |
-| allowed_by_methods | array[object] | False    |               | ["GET", "POST", "PUT", "DELETE", "PATCH", "HEAD", "OPTIONS", "CONNECT", "TRACE", "PURGE"] | List of allowed HTTP methods for a Consumer.                                   |
+| Name               | Type          | Required | Default       | Valid values  | Description |
+|--------------------|---------------|----------|---------------|---------------|-------------|
+| type               | string        | False    | consumer_name | ["consumer_name", "service_id", "route_id"]  | Type of object to base the restriction on.  |
+| allowed_by_methods | array[object] | False    |               | ["GET", "POST", "PUT", "DELETE", "PATCH", "HEAD", "OPTIONS", "CONNECT", "TRACE", "PURGE"] | List of allowed HTTP methods for a Consumer. |
```

Comment:
Hi 👋 May I know why this PR adjusts the order of the attributes? (The changes are not A-Z 😅)

Reply:
What do you mean by "A-Z"? 

I adjusted the order as I considered the priority of these three attributes. Please take a careful look at the `Description` of the `allowed_by_methods`, `whitelist` and `blacklist`.

Let me know if there are any concerns or suggestions. Thanks 🙏

Reply:
But the `required` fields are moved down, right? Developers may ignore those `required` fields IMO.

Reply:
👍 Good point, I didn't look at the `required` fields before.

Shall we just keep the original order?

Reply:
Yes, I prefer keeping `Required` fields in the top :)